challenges/dog-photo-gallery/script.js

```diff
@@ -0,0 +1,20 @@
+document.getElementById('btn-click').addEventListener('click', () => {
+    fetch("https://dog.ceo/api/breeds/image/random").then(Response => Response.json())
+    .then(data =>{
+        console.log(data)
+        let photoLink = data.message;
+
+        let ulElement = document.querySelector("#gallery");
+        let liNew = document.createElement("li");
+
+        liNew.innerHTML = `<img src="${photoLink}" class="li-img" alt="random dog">`;
```

Comment:
Here we're setting the `innerHTML` of this `li` to include literal text that we were handed by an API. This is problematic from a security perspective, as we don't trust the API response here. Have a read of https://medium.com/@gregadiaz89/dom-innerhtml-vulnerability-8821a03ef2b8 for some more context, and see if you can avoid this injection attack vulnerability :)